myuw/static/vue/components/cards/husky-experience.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <uw-card v-if="hxtViewer" :loaded="isReady" :errored="isErrored">
+    <template #card-heading>
+      <h3>
+        Husky Experience Toolkit
+      </h3>
+    </template>
+    <template #card-body>
+      <div v-html="article_html" />
+    </template>
+  </uw-card>
+</template>
+
+<script>
+import {mapGetters, mapState, mapActions} from 'vuex';
+import Card from '../../containers/card.vue';
+
+export default {
+  components: {
+    'uw-card': Card,
+  },
+  data: function() {
+    return {};
+  },
+  computed: {
+    ...mapState({
+      article_html: (state) => state.hx_toolkit.value.article_html,
+      hxtViewer: (state) => state.user.affiliations.hxt_viewer,
+    }),
+    ...mapGetters('hx_toolkit', {
+      isReady: 'isReady',
+      isErrored: 'isErrored',
+    }),
+  },
+  created() {
+    this.fetch();
```

Comment:
the fetch function should be only be called when hxtViewer is true so that we don't fetch data that we don't need